frontend/amundsen_application/static/js/config/config-utils.ts

```diff
@@ -145,6 +145,18 @@ export function getDisplayNameByResource(resourceType: ResourceType): string {
   return AppConfig.resourceConfig[resourceType].displayName;
 }
 
+/**
+ * Returns the displayName for the given resourceType
+ */
+export function getCustomDisplayNameByResource(
+  resourceType: ResourceType
+): string {
+  const resourceDisplayName = getDisplayNameByResource(resourceType);
+  return resourceDisplayName === 'Datasets'
+    ? 'Datasets / Fractal Features'
+    : resourceDisplayName;
+}
```

Comment:
I saw the `getDisplayNameByResource` are used in many places, I wonder if it will be easier to resolve conflict (if any) if we create this file in credit_card repo and overwrite the implementation of `getDisplayNameByResource`?